internal/cmd/group_utils.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tursodatabase/turso-cli/internal"
+	"github.com/tursodatabase/turso-cli/internal/turso"
+)
+
+func ensureGroupAwake(client *turso.Client, groupName string) (bool, error) {
+	group, err := getGroup(client, groupName)
+	if err != nil {
+		return false, fmt.Errorf("failed to get group info: %w", err)
+	}
+
+	groupStatus := aggregateGroupStatus(group)
+	if groupStatus != "Archived 💤" {
```

Comment:
I'm not a huge fan of checking a string with an emoji here, could lead to lots of issues. Do we need the emoji now that we've changed Sleeping to Archived where this was originally shown?

Reply:
Removed it